ImageKitIO.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'ImageKitIO'
-  s.version          = '3.0.0'
+  s.version          = '3.1.0'
```

Comment:
We should bump this to v4 and provide a changelog / migration guide to cover the following
- Breaking changes from v3.0
- Swift 6 concurrency requirements
- Code examples for common migrations